.jshintrc

```diff
@@ -23,6 +23,8 @@
   "laxcomma": true,
   "proto": true,
 
+  "esnext": true,
```

Comment:
we were missing some files in the linter, and removing this was actually too strict apparently.